test-case/check-alsabat.sh

```diff
@@ -95,6 +96,22 @@ function __upload_wav_file
     done
 }
 
+function check_wav_file_snr
+{
+    for file in /tmp/bat.wav.*
+    do
+	if test -s "$file"; then
+	    dlogi "Checking wav file $file"
+	    cd "$TESTDIR"/tools || {
```

Comment:
This seems overkill, `set -e` should be enough here. Why would this fail?

Reply:
Seems that I did it to avoid the shellcheck warning, but now it's removed.